### PR TITLE
Update swagger file to reflect specification text

### PR DESCRIPTION
**Pull request by koriaf**
and testpoint.io installation urls

Also fixed some minor problems with camelCase and wording

### CODE REVIEW

**Comment by koriaf** [comment]
@kshychko and @monkeypants, please check it out. Now it's more ausdigital.org spec compatible, but not completely (blocked by #3)

File: docs/1.0/swagger.json [hunk 1]
```diff
@@ -6,116 +6,99 @@
...
       "default":""
     },
-    "isRead":{
-      "name":"isRead",
+    "status":{
+      "name":"status",
       "in":"query",
-      "description":"TODO",
-      "required":true,
+      "description":"new|read, receiving status",
```
Comment (kshychko) [inline docs/1.0/swagger.json]: enum properyt needs to be added

File: docs/1.0/swagger.json [hunk 2]
```diff
@@ -196,7 +198,7 @@
...
+              "application/json":"{\n  \"links\":{\n      \"first\": \"https://tap-gw.testpoint.io/api/endpoints/?page=1\",\n      \"last\": \"https://tap-gw.testpoint.io/api/endpoints/?page=1\",\n      \"next\": null,\n      \"prev\": null\n  },\n  \"data\": [\n      {\n          \"type\": \"TapGwEndpoint\",\n          \"id\": \"XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXXX\",\n          \"attributes\":{\n              \"participantId\": \"urn:oasis:names:tc:ebcore:partyid-type:iso6523:0151::9999999999\",\n              \"endpointType\": \"ausdigital-tap/2\",\n              \"key1\": \"value1\",\n              \"key2\": \"value2\"\n          }\n      }\n  ],\n  \"meta\":{\n      \"pagination\":{\n          \"page\": 1,\n          \"pages\": 1,\n          \"count\": X\n      }\n  }\n}"
```
Comment (kshychko) [inline docs/1.0/swagger.json]: 'new' is not a valid JSON attribute value

File: docs/1.0/swagger.json [hunk 3]
```diff
@@ -667,15 +662,15 @@
...
-    "/api/messages/{messageId}/body/":{
+    "/messages/{messageId}/":{
       "get":{
-        "tags":[
+        "tags": [
           "Get TAP message body"
         ],
         "summary":"Get TAP message body",
-        "description":"Returns TAP message body",
+        "description":"Returns TAP message body (JSON file with fields like 'cyphertext' and so on, see `ausdigital-tap`)",
```
Comment (kshychko) [inline docs/1.0/swagger.json]: shall we use either ' or ` quotes? this is a JSON document, not an .md file, i think ' is preferable

**Comment by kshychko** [comment]
In general looks good to me, I can do the changes requested by me after PR is merged.
@monkeypants , please merge the PR, if you have no objections.

**Pull request merged**